File: study-android/src/main/java/com/pltech/study/android/MainActivity.java
package com.pltech.study.android;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.wb.study.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.btn_startActivity)
                .setOnClickListener(v -> {
                    Intent intent = new Intent(MainActivity.this, TargetActivity.class);
                    startActivity(intent);
                });
    }
}

File: study-android/src/main/java/com/wb/study/MainActivity.kt
package com.wb.study

import android.os.Bundle
import android.text.TextUtils
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil.setContentView
import com.wb.study.databinding.ActivityMainBinding


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.layout_test_view)
//        val mainBinding = setContentView(this, R.layout.activity_main) as ActivityMainBinding
//
//        val book = Book("Android 高质量编程指北", "Alex PL", 5)
//        mainBinding.setVariable(BR.book, book)
        //mainBinding.book = book

        //getVideoHeight(getLastImagePath())

        val test = "      "
        Log.d("TAG", "is test empty: " + TextUtils.isEmpty(test))
    }

}